docs/developer.md

```diff
@@ -195,28 +195,32 @@ sudo apt install miopen-hip hipfft-dev rocrand-dev hipsparse-dev hipsolver-dev \
     rccl-dev rccl hip-dev rocfft-dev roctracer-dev hipblas-dev rocm-device-libs
 ```
 
+The recommended way to install these dependencies is by running our script, `jax/build/rocm/tools/get_rocm.py`, 
+and selecting the appropriate options.
+
 To build jaxlib with ROCM support, you can run the following build command,
 suitably adjusted for your paths and ROCM version.
 
 ```
-python build/build.py --enable_rocm --rocm_path=/opt/rocm-5.7.0
+python3.10 ./build/build.py --use_clang=true --clang_path=/usr/lib/llvm-18/bin/clang-18 --enable_rocm --build_gpu_plugin --gpu_plugin_rocm_version=60 --rocm_path=/opt/rocm-6.2.3
```

Comment:
Just leave as `python3`

The clang path is going to be system dependent unfortunately, but I guess we could leave it up to the user to change the example path here.

```suggestion
python3 ./build/build.py --use_clang=true --clang_path=/usr/lib/llvm-18/bin/clang-18 --enable_rocm --build_gpu_plugin --gpu_plugin_rocm_version=60 --rocm_path=/opt/rocm-6.2.3
```